src/foam/support/model/messageModel.js

```diff
@@ -0,0 +1,29 @@
+foam.CLASS({
+    package: 'foam.support.model',
```

Comment:
Indent two spaces. See the FOAM style guide:
https://github.com/foam-framework/foam2/blob/master/doc/guides/StyleGuide.md